src/briefcase/integrations/android_sdk.py

```diff
@@ -282,52 +302,32 @@ def verify_install(
 """
                 )
 
+        # Verify Briefcase-managed Android SDK
         if sdk is None:
-            # Build an SDK wrapper for the Briefcase SDK instance.
             sdk_root_path = tools.base_path / "android_sdk"
             sdk = AndroidSDK(tools=tools, root_path=sdk_root_path)
 
-            if sdk.exists():
-                # NOTE: For now, all known versions of the cmdline-tools are compatible.
-                # If/when that ever changes, do a verification check here.
-                sdk.verify_license()
-            else:
-                # The legacy SDK Tools exist. Delete them.
-                if (sdk_root_path / "tools").exists():
-                    tools.logger.warning(
-                        f"""
-*************************************************************************
-** WARNING: Upgrading Android SDK tools                                **
-*************************************************************************
-
-    Briefcase needs to replace the older Android SDK Tools with the
-    newer Android SDK Command-Line Tools. This will involve some large
-    downloads, as well as re-accepting the licenses for the Android
-    SDKs.
+            if not sdk.exists():
+                if not install:
+                    raise MissingToolError("Android SDK")
 
-    Any emulators created with the older Android SDK Tools will not be
-    compatible with the new tools. You will need to create new
-    emulators. Old emulators can be removed by deleting the files
-    in {sdk.avd_path} matching the emulator name.
+                sdk.delete_legacy_sdk_tools()
 
-*************************************************************************
-"""
-                    )
-                    tools.shutil.rmtree(sdk_root_path)
+                tools.logger.info(
+                    "Upgrading Android SDK..."
+                    if sdk.cmdline_tools_path.parent.exists()
+                    else "The Android SDK was not found; downloading and installing...",
+                    prefix=cls.name,
+                )
+                tools.logger.info(
+                    "To use an existing Android SDK instance, specify its root "
+                    "directory path in the ANDROID_HOME environment variable."
+                )
```

Comment:
IMHO, if someone already has a working Briefcase install and is upgrading, there's no need to let them know that they can use an existing Android SDK instance. I think we can bury the "to use an existing" warning unless we're doing a first install.

Reply:
That makes sense. Details about `ANDROID_HOME` are only shown now for a new install.